types/common.ts

```diff
@@ -426,6 +426,8 @@ export type EntityType = "Corporation" | "LLC" | "Partnership" | "PubliclyTraded
 
 
 export interface RichMerchantData {
+    name?: string // Name of the merchant.
```

Comment:
```suggestion
    name: string // Name of the merchant.
```

This matches the type from unit